### ISSUE TITLE
Add a search block

### ISSUE THREAD

**Issue by elessar-ch** [axa-ch/style-guide#332]
Add a search block or extend the autocomplete block to fully support the search use case.
The main difference from our existing autocomplete is the magnifier icon.

In this case I would extend the autocomplete block. @davidknezic What do you think?

**Comment by davidknezic** [comment]
I agree. Let's extend the `autocomplete` block and rename it to `search`.

**Comment by elessar-ch** [comment]
@martinoss You had asked about a search block too. Do you have any inputs?

**Comment by sventschui** [comment]
> Let's extend the `autocomplete` block and rename it to `search`.

Would rather stay with `autocomplete` and add a `search` modifier instead...
